### PR TITLE
Update URIs to be https and drop jsonld

### LINKED ISSUE #329: Change structure of vocab URLs
Make `https` and remove `.jsonld`. 

For example:
```
https://cocina.sul.stanford.edu/models/admin_policy
```

### PR DESCRIPTION
**Pull request by jcoyne**
**NOTE:  Changes to openapi.yml require updating openapi.yml for sdr-api and dor-services-app and generating models - see README.**

## Why was this change made? 🤔
Fixes #329

## How was this change tested? 🤨

⚡ ⚠ If this change has cross service impact, ***run [integration tests](https://github.com/sul-dlss/infrastructure-integration-test)*** and/or test in [stage|qa] environment, in addition to specs. ⚡

### CODE REVIEW

File: lib/cocina/models/collection.rb [hunk 1]
```diff
@@ -7,11 +7,11 @@ class Collection < Struct
...
-      TYPES = ['http://cocina.sul.stanford.edu/models/collection.jsonld',
-               'http://cocina.sul.stanford.edu/models/curated-collection.jsonld',
-               'http://cocina.sul.stanford.edu/models/user-collection.jsonld',
-               'http://cocina.sul.stanford.edu/models/exhibit.jsonld',
-               'http://cocina.sul.stanford.edu/models/series.jsonld'].freeze
+      TYPES = ['https://cocina.sul.stanford.edu/models/collection',
+               'https://cocina.sul.stanford.edu/models/curated-collection',
+               'https://cocina.sul.stanford.edu/models/user-collection',
+               'https://cocina.sul.stanford.edu/models/exhibit',
+               'https://cocina.sul.stanford.edu/models/series'].freeze
```
Comment (mjgiarlo) [inline lib/cocina/models/collection.rb:14]: If we remove the old URIs from these lists right now, won't lots of objects be confused as to what they are?

**Review by mjgiarlo — COMMENTED** [review]
I have a question about existing type URIs being removed, specifically about the `Checkable` module's downstream consumers: DSA in particular asks Cocina objects whether they are DROs or Collections (etc.) and if the old-style type URIs are persisted but the new-style ones are in the gem, DSA is going to be confused. (Right?)

File: lib/cocina/models/collection.rb (hunk 1 above)
Comment (jcoyne) [reply lib/cocina/models/collection.rb:14]: No, because we don't persist these in Fedora yet. That's why it's important to make these changes now.

**Review by mjgiarlo — APPROVED** [review]
🚀

**Pull request merged**